Makefile

```diff
@@ -63,3 +63,44 @@ endif
 cover-html: test-unit-cover
 	@echo "--> Opening in the browser"
 	@go tool cover -html=$(TEST_COVERAGE_PROFILE)
+
+###############################################################################
+##                                Infrastructure                             ##
+###############################################################################
+
+BITCOIND_SNAPSHOT=$(shell pwd)/contrib/snapshot
+BITCOIND_DATA=$(shell pwd)/contrib/regtest
+
+run-bitcoind:
+	@docker run -v $(BITCOIND_DATA):/bitcoin/.bitcoin --name=bitcoind-node -d \
+        -p 18444:8333 \
+        -p 127.0.0.1:18443:8332 \
+        -e REGTEST=1 \
+        -e DISABLEWALLET=0 \
+        -e PRINTTOCONSOLE=1 \
+        -e RPCUSER=mysecretrpcuser \
+        -e RPCPASSWORD=mysecretrpcpassword \
+        kylemanna/bitcoind
```

Comment:
let's define a version here to assure we all use the same version:
```
BITCOIND_CONTAINER="kylemanna/bitcoind:2400e6496045"
```

Reply:
Done!